src/midi/midi_coremidi.h

```diff
@@ -22,20 +22,27 @@
 #ifndef DOSBOX_MIDI_COREMIDI_H
 #define DOSBOX_MIDI_COREMIDI_H
 
+#include "midi_handler.h"
+
 #include <CoreMIDI/MIDIServices.h>
 #include <sstream>
 #include <string>
 
+#include "programs.h"
+
 class MidiHandler_coremidi : public MidiHandler {
 private:
 	MIDIPortRef m_port;
 	MIDIClientRef m_client;
 	MIDIEndpointRef m_endpoint;
 	MIDIPacket* m_pCurPacket;
 public:
-	MidiHandler_coremidi()  {m_pCurPacket = 0;}
-	const char * GetName(void) { return "coremidi"; }
-	bool Open(const char * conf) {	
+	MidiHandler_coremidi() { m_pCurPacket = 0; }
```

Comment:
Can this member assignment during construction `{ m_pCurPacket = 0; }` be done directly at the member definition?

Reply:
I added one new commit to the PR to correctly initialize bot macOS-specific MIDI handlers :)